odxtools/communicationparameter.py

```diff
@@ -76,6 +76,9 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return self.__repr__()
 
+    def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
+        return {}
+
```

Comment:
This PR is already too big, throwing in small random changes like this one is not helping

Reply:
right. the issue with things like this is that you stumble over them while working on the actual subject of the PR, you fix it (maybe because it simplifies things elsewhere, in this case, this avoids needing to treat special treatment of the comparams in `DiagLayer._build_odxlinks()`) and then I promptly forget. I'll factor this out into a separate commit and open a PR for it. (NB: that the PR here is already an effort of getting a substantially larger patch into a more digestible size...)

Reply:
okay, the outsourced PR is #134. (i had to rebase this one on #134 to avoid a trivial cleanup PR after both have merged.)